### ISSUE TITLE
Add additional error message for invalid peers.json

### ISSUE THREAD

**Issue by Tjstretchalot** [rqlite/rqlite#1217]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when creating a peers.json and I use `http` which causes esoteric errors like "too many colons in address"

**Describe the solution you'd like**
If peers.json has http at the start of any of the addresses, emit a clear error message that it should not be a url, but instead just an ip address

**Describe alternatives you've considered**
- Be more clear in documentation - might not hurt, but it's a fairly subtle issue
- Just detect http and then handle it correctly. I think it's best to defer to "there should be only one right way" in this context

**Additional context**
This has happened to me 3 times and each time it took me on a while goose chase because of all the various unrelated error messages. Each time I realize it's labelled address not url, but nonetheless I think a simple error message is going to be more effective than berating my memory!

**Comment by otoolep** [comment]
Just so we're clear, can you show me an example of peers.json that causes you problems? Also, just curious why you use it? It's mostly a recovery-only thing, are your clusters failing?

**Comment by Tjstretchalot** [comment]
I use if for recovery typically after I accidentally mess up and turn two instances offline, since usually no errors occur if there was an issue with the first node coming online (as the whole point is that the cluster can handle 1 node down out of 3), but when the second node gets replaced then I need to go in and do recovery as now I've lost 2/3 nodes.

One of my absolute favorite parts of rqlite is just how easy the recovery process is in general - it's way less stressful compared to e.g. my past experience trying to handle postgresql recovery.

For more context, I try to keep my rqlite instances at the bleeding edge, hence they pull the latest rqlite release and the latest amazon AMI approximately once every 2 weeks. To do this, the best route I've come up with as follows:

- Suppose I have nodes A, B, C
- Take node A down, spin up node D with the same rqlite id as A had, let it recover.
- Take node B down, spin up node E with the same rqlite id as B had, let it recover.
- Take node C down, spin up node F with the same rqlite id as C had, let it recover

I just switched to the [automatic bootstrapping](https://rqlite.io/docs/clustering/automatic-clustering/#automatic-bootstrapping) which I expect will make this process a lot easier - before, it was almost always a problem where if the first or second node was the configured master node it would end up starting as a single-node instance rather than joining the cluster, and then I take the second node offline and the whole thing is borked, leading to recovery.  Now with automatic bootstrapping being able to tell it to expect a cluster, I imagine I won't need to use peers.json nearly as often!

Nonetheless, I still think detecting if an http prefix was incorrectly added would be an easy quality of life win. This is an example of a peers.json that causes the issue:

```json
[
  {
    "id": "1",
    "address": "http://10.0.2.16:4002",
    "non_voter": false
  },
  {
    "id": "2",
    "address": "http://10.0.4.23:4002",
    "non_voter": false
  },
  {
    "id": "3",
    "address": "http://10.0.6.75:4002",
    "non_voter": false
  }
]
```

---

The other thing that can be challenging with constructing these peers.json is I can never remember if it wants 4001 (the http address) or 4002 (the raft address). However, I don't expect it to help much there considering I don't know a good way for it to detect if I selected the wrong one.
